central/networkgraph/flow/datastore/flow_impl.go

```diff
@@ -125,6 +127,10 @@ func (fds *flowDataStoreImpl) RemoveFlowsForDeployment(ctx context.Context, id s
 }
 
 func (fds *flowDataStoreImpl) RemoveMatchingFlows(ctx context.Context, keyMatchFn func(props *storage.NetworkFlowProperties) bool, valueMatchFn func(flow *storage.NetworkFlow) bool) error {
+	if env.PostgresDatastoreEnabled.BooleanSetting() {
```

Comment:
Gee, we should get ready to remove these soon.

Reply:
Right and all the rocks code